app/models/system_console.rb

```diff
@@ -53,10 +53,10 @@ def self.launch_proxy(remote_address, remote_port)
       return nil
     end
 
-    command = '/usr/bin/socat', "TCP-LISTEN:#{local_port},fork", "TCP:#{remote_address}:#{remote_port}"
+    command = ["/usr/bin/socat", "TCP-LISTEN:#{local_port},fork", "TCP:#{remote_address}:#{remote_port}"]
     _log.info("Running socat proxy command: #{command.join(' ')}")
 
-    pid = spawn(*command)
+    pid = spawn(AwesomeSpawn::CommandLineBuilder.new.build(command[0], command[1..-1]))
```

Comment:
I understand that brakeman likes this new syntax. But does this actually protect us?

Reply:
Yes....awesome spawn creates safe command lines.